### PR TITLE
IGVF-1016-standardize-assembly

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

**Review by gabdank — CHANGES_REQUESTED** [review]
on many objects (like allignment file, gene) you remove hg19, mm9, mm10.. but the upgrade is not taking care of potential cases were these assemblies were used. I think you have to assume some objects used "old" assemblies and handle that gracefully (adding notes and changing to valid assembly) although we definitly have to try and clean up wrong assemblies by wrangling before this ticket gets in

File: src/igvfd/schemas/construct_library_set.json [hunk 1]
```diff
@@ -562,7 +562,7 @@
     },
     "properties": {
         "schema_version": {
-            "default": "4"
+            "default": "6"
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/construct_library_set.json:565]: The schema wasn't bumped from the previous upgrade of 4->5.

File: src/igvfd/schemas/prediction_set.json [hunk 2]
```diff
@@ -214,7 +214,7 @@
     },
     "properties": {
         "schema_version": {
-            "default": "4"
+            "default": "6"
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/prediction_set.json:217]: The schema wasn't bumped from the previous upgrade of 4->5.

File: src/igvfd/schemas/changelogs/alignment_file.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 ## Changelog for *`alignment_file.json`*
 
+### Schema version 6
+
+* Add `transcriptome_annotation`.
+* Remove `hg19` and `mm10` from `assembly`.
+* Move `assembly` to mixins.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/alignment_file.md:7]: removed

File: src/igvfd/schemas/changelogs/gene.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`gene.json`*
 
+### Schema version 7
+
+* Remove `hg19`, `mm10`, and `mm9` from `assembly` on `locations`.
+* Move `assembly` on `locations` to mixins.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/gene.md:6]: removed

File: src/igvfd/schemas/changelogs/genome_browser_annotation_file.md [hunk 5]
```diff
@@ -1,5 +1,11 @@
 ## Changelog for *`genome_browser_annotation_file.json`*
 
+### Schema version 6
+
+* Add `GENCODE 44`, `GENCODE 45`, `GENCODE M33`, and `GENCODE M34` to `transcriptome_annotation`.
+* Remove `hg19` and `mm10` from `assembly`.
+* Move `assembly` to mixins.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/genome_browser_annotation_file.md:7]: removed

File: src/igvfd/schemas/changelogs/human_genomic_variant.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 3
 
+* Move `assembly` to mixins.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/human_genomic_variant.md:5]: removed

File: src/igvfd/schemas/changelogs/reference_file.md [hunk 7]
```diff
@@ -3,6 +3,8 @@
 ### Minor changes since schema version 10
 
 * Add `elements reference` to `content_type`.
+* Add `GENCODE 44`, `GENCODE 45`, `GENCODE M33`, and `GENCODE M34` to `transcriptome_annotation`.
+* Move `assembly` to mixins.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/reference_file.md:7]: removed

File: src/igvfd/schemas/changelogs/signal_file.md [hunk 8]
```diff
@@ -1,5 +1,11 @@
 ## Changelog for *`signal_file.json`*
 
+### Schema version 6
+
+* Add `transcriptome_annotation`.
+* Remove `hg19` and `mm10` from `assembly`.
+* Move `assembly` to mixins.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/signal_file.md:7]: removed

File: src/igvfd/schemas/changelogs/tabular_file.md [hunk 9]
```diff
@@ -1,5 +1,11 @@
 ## Changelog for *`tabular_file.json`*
 
+### Schema version 6
+
+* Add `GENCODE 44`, `GENCODE 45`, `GENCODE M33`, and `GENCODE M34` to `transcriptome_annotation`.
+* Remove `hg19` and `mm10` from `assembly`.
+* Move `assembly` to mixins.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/tabular_file.md:7]: removed

File: src/igvfd/schemas/changelogs/variant.md [hunk 10]
```diff
@@ -1,5 +1,6 @@
 ## Changelog for variant.json
 
+* Move `assembly` to mixins. (03/05/2024)
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/variant.md:3]: removed

**Review by gabdank — DISMISSED** [review]

**Pull request merged**